src/Fixer/Whitespace/NoExtraBlankLinesFixer.php

```diff
@@ -441,13 +441,7 @@ private function removeEmptyLinesAfterLineWithTokenAt(int $index): void
                 continue;
             }
 
-            $pos = strrpos($content, "\n");
```

Comment:
is this a clean up or can we maybe add a utest for this change to NoExtraBlankLinesFixerTest as well?

Reply:
This is only a refactoring that shouldn't change behavior.